langroid/agent/base.py

```diff
@@ -1109,65 +1164,18 @@ def tool_validation_error(self, ve: ValidationError) -> str:
             [f"{e['loc']}: {e['msg']}" for e in ve.errors() if "loc" in e]
         )
         return f"""
-        There were one or more errors in your attempt to use the 
-        TOOL or function_call named '{tool_name}': 
+        There were one or more errors in your attempt to use the
+        TOOL or function_call named '{tool_name}':
         {bad_field_errors}
         Please write your message again, correcting the errors.
         """
 
-    def handle_message(
-        self, msg: str | ChatDocument
-    ) -> None | str | OrderedDict[str, str] | ChatDocument:
+    def _check_multiple_orch_tools(
```

Comment:
Perhaps call it `_get_multiple_orch_tool_errs` or similar given the return type.

Reply:
agreed - changing